types/nightwatch/index.d.ts

```diff
@@ -5811,12 +5865,17 @@ export interface WebDriverProtocolCommandContexts {
      *    browser.windowMaximize('current', function(result) {
      *      console.log(result);
      *    });
-     * }
+     *  }
+     *
+     * @see https://nightwatchjs.org/api/windowMaximize.html
      */
+    windowMaximize(
+        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<null>) => void,
+    ): Awaitable<this, null>;
```

Comment:
Even with just callback, it will work as shown in example here https://nightwatchjs.org/api/windowMaximize.html#apimethod-container right?

Reply:
Yeah, it will work but not as expected as the callback won't be called. It will not throw any error and the window will maximize, but if you try to do something in the callback, that won't get executed as the callback is never called.